src/commands/support/manualleg.ts

```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/752801628347957248/929071827752480839/mcdu-discontinuity-manual-conceptual.png';
```

Comment:
```suggestion
const MANUAL_LEG_IMAGE_URL = 'https://cdn.discordapp.com/attachments/752801628347957248/929071827752480839/mcdu-discontinuity-manual-conceptual.png';
```

Clears up constant naming. 'MANUAL' could be mistaken for a traditional manual etc when searching the repo.